src/plugins/intel_gpu/src/graph/broadcast.cpp

```diff
@@ -69,7 +69,7 @@ broadcast_inst::typed_primitive_inst(network& network, broadcast_node const& nod
     size_t index = 0;
     size_t input_index = broadcast_axes_size;
 
-    if (format == format::bfzyx) {
+    if (format == format::bfzyx || format == format::b_fs_zyx_fsv16 || format == format::b_fs_zyx_fsv32 || format == format::bs_fs_zyx_bsv16_fsv16) {
```

Comment:
I think all of these stuffscan be simplified something like this: 

> auto max_axes_num = input_layout.get_rank();
   OPENVINO_ASSERT(axis>= 0 && axis < max_axes_num, "Incorrect parameters configuration: broadcast_axes ....");